site/src/pages/TerminalPage/TerminalPage.test.tsx

```diff
@@ -38,7 +38,7 @@ Object.defineProperty(window, "TextEncoder", {
 const renderTerminal = async (
   route = `/${MockUser.username}/${MockWorkspace.name}/terminal`,
 ) => {
-  const utils = renderWithAuth(<TerminalPage />, {
+  const utils = renderWithAuth(<TerminalPage renderer="dom" />, {
```

Comment:
Is this because Jest can't handle canvas elements? If so, how was this test working before? Asking because I'm curious if this canvas switch is the culprit behind #7948.

Reply:
Yup the Jest test looks for HTML elements which will only be present in the DOM renderer.  Technically setting `renderer` here is not necessary since the default is already DOM but I added it be explicit.

Before the major bump to xterm, back when the default was canvas, we would explicitly set `rendererType` to `dom` so it would have been using the DOM renderer then too.

Unfortunately I think that means #7948 is unrelated.  If it was using canvas I think we would get a different error since `.xterm-rows` should not exist at all (we should have seen the error `no xterm-rows` instead).